test/e2e/support/utils.js

```diff
@@ -196,3 +196,14 @@ export async function publishPost() {
 	// A success notice should show up
 	return page.waitForSelector( '.notice-success' );
 }
+
+/**
+ * Clicks on the button in the header which opens Document Settings sidebar when it is closed.
+ */
+export async function ensureDocumentSettingsSidebarIsOpened() {
```

Comment:
Not certain if this is the best name for the function, because if Publish panel is open, the button is not rendered and the function will not be able to ensure "document settings sidebar is opened".
An option is call the function click toggle sidebar or something similar but off course that name will bot transmit the main objective of clicking in the sidebar.

Reply:
Yes openDocumentSettingsSidebar looks good, as it is not strong as "ensure".